source/includes/_payment.md

```diff
@@ -2312,9 +1461,6 @@ class Payment {
    "auditLogs":[]
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?